quicklogic/common/utils/testarch_graph.py

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/env python3
```

Comment:
Hmmm, I don't recall this script being needed for QuickLogic PP3 architecture. If it is not used anywhere and is not an important utility we might remove it.

Reply:
I don't see it being used anywhere in the codebase so I removed it